frontend/src/static/js/v_segment.js

```diff
@@ -9,7 +9,10 @@ window.vSegment = {
   methods: {
     loadCode() {
       this.loaded = true;
-      window.toggleNext(this.$el.childNodes[0]);
+      // For updating buttons only once code has loaded into DOM
```

Comment:
Can be `Displaying code block only after the code has loaded into DOM`

Reply:
Indeed, I overlooked clarifying between dom / css updates. Thank you!

> Can be `Displaying code block only after the code has loaded into DOM`

Shouldn't this still be `For updating buttons only once code has loaded into DOM` though?
Since as you mentioned, the css update (buttons) follows the secondary dom update (v-for code)

Reply:
It is just a suggestion. Actually, the CSS update not only updates the button but also controls the code block to be displayed, which is the major functionality of this CSS update. That's the reason I give the above recommendation.